admin/broadleaf-contentmanagement-module/src/main/resources/config/bc/jpa/domain/URLHandler.orm.xml

```diff
@@ -39,5 +39,25 @@
         </query>
     </named-query>
 
+    <named-query name="BC_READ_BY_DESTINATION_URL">
+        <query>SELECT handler
+            FROM org.broadleafcommerce.cms.url.domain.URLHandler handler
+            WHERE
+            handler.newURL = :newURL
+        </query>
+    </named-query>
```

Comment:
We generally want to use `TypedQueries` like the following (https://cl.ly/351q262j3d14) instead of these orm queries